spyderlib/widgets/externalshell/sitecustomize.py

```diff
@@ -324,8 +325,6 @@ def qt_nt_inputhook():
 #==============================================================================
 # Matplotlib settings
 #==============================================================================
-IS_IPYTHON = os.environ.get("IPYTHON_KERNEL", "").lower() == "true"
-
 if matplotlib is not None:

```

Comment:
@PierreRaybaut, your change will break this entire block, which is needed to set the Matplotlib backend in our Python consoles (in Windows and Linux).